### PR TITLE
include test for zero

### PR DESCRIPTION
**Pull request by henrikeklund**
I thought it was a smart idea to bla bla black

### CODE REVIEW

**Comment by Reyna-RT** [comment]
Dear Henke

There is a bu-bu in your test, please fix

File: test_workshop.py [hunk 1]
```diff
@@ -23,3 +23,9 @@ def test_complex():
 	obs = mean(num_list)
 	exp = NotImplemented
 	assert obs == exp
+
+def test_zero():
+	num_list = [0,2,4,6]
+	obs = mean(num_list)
+	exp = 0
```
Comment (Reyna-RT) [inline test_workshop.py]: it should be `exp = 3`
